package.json

```diff
@@ -221,7 +222,7 @@
     "docker-toolbelt": "^3.3.8",
     "dockerode": "^2.5.8",
     "ejs": "^3.1.3",
-    "etcher-sdk": "^2.0.20",
+    "etcher-sdk": "^5.2.2",
```

Comment:
Looks like we now have v6.2.0 available:
https://github.com/balena-io-modules/etcher-sdk/blob/master/CHANGELOG.md

Reply:
@pdcastro sure, I'll add it as soon as we merge the balena-device-init pr. I'll add this ^ and replace the npm pre-releases.